monitoring/scripts/track-cid-backfill/src/queries.ts

```diff
@@ -0,0 +1,97 @@
+import { QueryTypes } from "sequelize";
+import { sequelizeConn } from "./db";
+
+import fs from 'fs-extra'
+
+// Fetch all content nodes
+export async function getAllContentNodes(): Promise<
+  { spid: number; endpoint: string }[]
+> {
+  const endpointsResp: unknown[] = await sequelizeConn.query(
+    `
+    SELECT cnode_sp_id as spid, endpoint
+    FROM ursm_content_nodes
+    WHERE is_current = TRUE; 
+  `,
+    {
+      type: QueryTypes.SELECT,
+    }
+  );
+
+  const endpoints = (endpointsResp as { endpoint: string; spid: number }[])!;
+
+  return endpoints;
+}
+
+export async function getTrackCount() {
+  const countResp: unknown[] = await sequelizeConn.query(
+    `
+    SELECT COUNT(*) 
+    FROM tracks 
+    WHERE is_current = TRUE
+    AND is_delete = FALSE
+    AND track_cid is NULL; 
+  `,
+    {
+      type: QueryTypes.SELECT,
+    }
+  );
+
+  const count = parseInt((countResp[0] as { count: string }).count);
+
+  return count;
+}
+
+// returns trackIds 
+export async function getTrackIdBatch(
+  offset: number,
+  batchSize: number
+): Promise<number[]> {
+  console.log(`getting ${batchSize} items at offset ${offset}`);
+  const batchResp: unknown[] = await sequelizeConn.query(
+    `
+    SELECT track_id
+    FROM tracks
+    WHERE is_current = TRUE
+    AND is_delete = FALSE
```

Comment:
we don't care about is_delete = false?

Reply:
I'll remove it. Should I regenerate the staging csv?

Reply:
but lets handle deleted for prod as well

Reply:
yup, already did